### ISSUE TITLE
Re: Mouse Over information box for WAITING LIST button

### ISSUE THREAD

**Issue by ronenreinheimer** [206Tours/DB#1967]
Hi Ronen,

The ,css and .js files updated, but you'll need to update the php code for this section.

Brad, 

Please check the screenshot to check changes that you'll need to make - https://pastenow.ru/cd8d976d208d4ac3f18073b06ae3ac4a

Thanks,
Andrew

**Comment by ronenreinheimer** [comment]
hi Brad,

i see a waiting list button on tour111:
http://www.206tours.com/tour111/

is it supposed to be working here?

**Comment by bradatkins** [comment]
No, because I previously asked what exactly am I showing in the mouse over?

**Comment by bradatkins** [comment]
Ronen, what do you want to show when the user mouses over?

**Comment by ronenreinheimer** [comment]
Can it be something that we enter into a text field in the DB? something we can control in case we want to change it?

**Comment by bradatkins** [comment]
Of course it can, but it doesn't exist today...
